api/app/src/mappings/google/models/biometrics.ts

```diff
@@ -0,0 +1,16 @@
+import { z } from "zod"
+import { googleResp } from ".";
+
+export const googleBiometricsDataSourceId = z.enum([
+  "derived:com.google.blood_pressure:com.google.android.gms:merged",
+  "derived:com.google.blood_glucose:com.google.android.gms:merged",
+  "derived:com.google.body.temperature:com.google.android.gms:merged",
+  "derived:com.google.oxygen_saturation:com.google.android.gms:merged",
+  "derived:com.google.heart_rate.bpm:com.google.android.gms:merge_heart_rate_bpm",
```

Comment:
General comment: these string values in the models are duplicated in the consts in the mappings. I'd define these consts once in the models, and use them in the mappings as well.